packages/story-editor/src/app/story/actions/useAutoSave.js

```diff
@@ -39,7 +39,7 @@ function useAutoSave({ storyId, pages, story }) {
   const {
     actions: { autoSaveById },
   } = useAPI();
-  const { metadata } = useConfig();
+  const { metadata, flags } = useConfig();
```

Comment:
Why use flags like this? Why can't we use the flagged provider?

Reply:
Because `getStoryMarkup()` (which generates the output) is just a simple function, not a hook or React component